src/services/AdvancedTradingView/useAllTrades.hook.ts

```diff
@@ -60,6 +60,10 @@ export const useAllTrades = (): {
             transaction.status.toUpperCase() !== TransactionStatus.REDEEM
         )
         .splice(0, 5 - pendingTransactions.length)
+        // Sort by conirmation time
```

Comment:
```suggestion
        // Sort by confirmation time
```

Reply:
Addressed in https://github.com/ImpeccableHQ/swapr-dapp/pull/1646/commits/83426e029f16a9fc9ce601d3cecea919e55cd64f